src/Simulator/Simulator.h

```diff
@@ -55,6 +55,12 @@ class cSimulator
 	friend class cChunk;  // Calls AddBlock() in its WakeUpSimulators() function, to speed things up
 
 	/** Called to simulate a new block */
+	virtual void AddBlock(Vector3i a_Block, cChunk * a_Chunk)
+	{
+		AddBlock(a_Block.x, a_Block.y, a_Block.z, a_Chunk);
+	}
+
+	/** Called to simulate a new block (DEPRECATED in favor of vector-parametered version) */
 	virtual void AddBlock(int a_BlockX, int a_BlockY, int a_BlockZ, cChunk * a_Chunk) = 0;
```

Comment:
So can we get rid of this overload now? It seems everything is ready for the change.